src/builtin.c

```diff
@@ -1672,27 +1703,21 @@ static jv f_current_line(jq_state *jq, jv a) {
 
 #define LIBM_DD(name) \
   {f_ ## name,  #name, 1},
-#define LIBM_DD_NO(name)
+#define LIBM_DD_NO(name) LIBM_DD(name)
```

Comment:
This took longer to understand that i want to admit :) too early here

Reply:
^^